corebehrt/main/pretrain.py

```diff
@@ -38,8 +38,10 @@ def main_train(config_path):
     train_dataset, val_dataset = DatasetPreparer(cfg).prepare_mlm_dataset()
 
     if "scheduler" in cfg:
```

Comment:
This is duplicated for finetune, so perhaps it should be wrapped in a functionality for standardization?

Reply:
I'm not sure what you mean exactly. You mean the whole thing with the if statement?

Reply:
Yes. If we have to adjust the scheduler in the same way for both pretraining and finetune, we should standardize it, rather than having it duplicates in both scripts

Reply:
But now where I look at it today, I guess it's just an if_statement with a log, so might not be needed